src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1662,7 +1662,7 @@ public function testTrustedProxies()
         // trusted proxy via setTrustedProxies()
         Request::setTrustedProxies(array('3.3.3.3', '2.2.2.2'));
         $this->assertEquals('1.1.1.1', $request->getClientIp());
-        $this->assertEquals('real.example.com', $request->getHost());
+        $this->assertEquals('foo.example.com', $request->getHost());
```

Comment:
so technically the behaviour is changed here for handling the `X_FORWARDED_HOST` header? Does it matter for BC?